common/js/src/logging/debug-dump.js

```diff
@@ -187,13 +187,13 @@ GSC.DebugDump.dump = function(value) {
   // produce thorny false positives.
   //
   // TODO(emaxx): Think about a proper solution that deals with cyclic references.
-  if (value instanceof Document)
+  if (Document && value instanceof Document)
```

Comment:
Haha interesting behavior. I was about to write "maybe put a if (!value) return 'undefined';" when I realized the right side is what's undefined. Strange indeed.